e2e/fixtures/base/package.json

```
@@ -0,0 +1,19 @@
+{
+  "name": "fixture-base",
+  "type": "module",
+  "description": "Basic tuono application",
+  "version": "0.0.1",
+  "scripts": {
+    "test:e2e": "pnpm exec playwright test"
```

`pnpm exec` should not be needed inside package.json scripts

```suggestion
    "test:e2e": "playwright test"
```

---

Updated 👍